swagger.yaml

```diff
@@ -383,6 +383,30 @@ paths:
       responses:
         '204':
           description: Always responds with 204 code
+  '/containers/info':
+    post:
+      description: >
+        This Endpoint returns information about products attached to containers 
+      produces:
+        - application/json
+      tags:
+        - WidgetContainer
+      parameters:
+        - name: widget_container_uuids
+          in: query
+          description: Filters Containers by Array of UUIDs
+          required: false
+          type: array
+        - name: has_feedback
+          in: query
+          description: Returns only Containers with Feedbacks
+          required: false
+          type: boolean
+      responses:
+        '200':
+          description: Succssful response
+          # schema:
```

Comment:
@asux hi alex, 
can you tell me how I add a response schema to the documentation